.github/workflows/screener-run.yml

```diff
@@ -0,0 +1,183 @@
+name: Screeners workflow
+
+on: workflow_dispatch
```

Comment:
I think that even in this PR you can make this workflow depend already on the screener build workflow